examples/src/bin/lcd_cam_ov2640.rs

```diff
@@ -0,0 +1,422 @@
+//! Drives the camera on a Freenove ESP32-S3 WROOM
```

Comment:
This is the exact same pin layout on the esp32s3-eye, so would you mind documenting that?

Reply:
Yeah sure! A lot of dev boards seem to share pin layouts. I'm guessing it's to make examples portable.